nautobot/ipam/models.py

```diff
@@ -502,8 +519,12 @@ class Meta:
             "namespace",
             "network",
             "prefix_length",
-        )  # (vrf, prefix) may be non-unique
-        unique_together = ("namespace", "network", "prefix_length")
+        )
+        index_together = [
+            ["network", "broadcast", "prefix_length"],
+            ["namespace", "network", "broadcast", "prefix_length"],
+        ]
+        unique_together = ["namespace", "network", "prefix_length"]
```

Comment:
Should be able to remove the `natural_key` `AttributeRemover` from this model now that it has a proper natural key. Yay!